src/routes/(app)/order/[id]/fetchOrderForUser.ts

```diff
@@ -135,6 +136,7 @@ export async function fetchOrderForUser(orderId: string) {
 				isEmployee: note.role !== CUSTOMER_ROLE_ID,
 				alias: note.userAlias
 			})) || [],
-		receiptNote: order.receiptNote
+		receiptNote: order.receiptNote,
+		user: pojo(order.user)
```

Comment:
Just send the user.email & npub if that's all that's needed, we don't want to send `sessionId`